### PR TITLE
Add WebAppsWG boilerplates

### PR DESCRIPTION
**Pull request by marcoscaceres**
Group is launching this week, so we are moving specs over. 

@tabatkins, let me know know if I've missed anything. I left the old test files alone, as I figure they are fine even though the point to the old group.

### CODE REVIEW

File: bikeshed/boilerplate/webapps/header.include [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <title>[TITLE]</title>
+  <link href="https://www.w3.org/StyleSheets/TR/2016/W3C-[STATUS]" rel=stylesheet type="text/css">
```
Comment (tabatkins) [inline bikeshed/boilerplate/webapps/header.include]: Use the [W3C-STYLESHEET] macro for the whole link instead; the stylesheet urls don't all follow this pattern.

File: bikeshed/config/status.py [hunk 2]
```diff
@@ -86,7 +86,7 @@
 noEDStatuses = ["LS", "LS-COMMIT", "LS-BRANCH", "LD", "FINDING", "DRAFT-FINDING", "DREAM", "iso/NP", "whatwg/RD"]
 
 megaGroups = {
-    "w3c": frozenset(["act-framework", "audiowg", "csswg", "dap", "fxtf", "fxtf-csswg", "geolocation", "houdini", "html", "i18n", "immersivewebwg", "mediacapture", "processcg", "ricg", "sacg", "serviceworkers", "svg", "texttracks", "uievents", "wasm", "web-bluetooth-cg", "webappsec", "webauthn", "webml", "web-payments", "webperf", "webplatform", "webrtc", "webspecs", "webvr", "wicg"]),
+    "w3c": frozenset(["act-framework", "audiowg", "csswg", "dap", "fxtf", "fxtf-csswg", "geolocation", "houdini", "html", "i18n", "immersivewebwg", "mediacapture", "processcg", "ricg", "sacg", "serviceworkers", "svg", "texttracks", "uievents", "webapps", "wasm", "web-bluetooth-cg", "webappsec", "webauthn", "webml", "web-payments", "webperf", "webplatform", "webrtc", "webspecs", "webvr", "wicg"]),
```
Comment (tabatkins) [inline bikeshed/config/status.py]: *so close* to alphabetical, but still not quite right ^_^

Comment (marcoscaceres) [reply bikeshed/config/status.py]: Heh, busted 😅

**Comment by marcoscaceres** [comment]
> Please delete the .DS_Store files from your PR.

Will do. Should I send a new PR for this tho? The `.DS_Store` files show up all over my tree on MacOS, so worried I'll accidentally commit one.

File: bikeshed/boilerplate/webapps/header.include (hunk 1 above)
Comment (marcoscaceres) [reply bikeshed/boilerplate/webapps/header.include]: I'll fix this one up. Should I send a separate PR for webplatform? (this was just copy/pasta from the webplatform folder)

**Comment by marcoscaceres** [comment]
Thanks for the speedy review, @tabatkins. I'm unsure of BikeShed's release cadence, but if this could be prioritized then it would help us republish all the Web Apps WG specs with the updated boilerplate.

**Comment by tabatkins** [comment]
Lol I thought I merged it already, thanks for the ping

**Pull request merged**